test/jdk/sun/security/mscapi/AllTypes.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Do not remove the old year. This should be "Copyright (c) 2022, 2024, Oracle..."